src/pybind/mgr/dashboard/frontend/src/app/ceph/cephfs/cephfs-mount-details/cephfs-mount-details.component.html

```diff
@@ -1,37 +1,47 @@
 <cds-modal size="md"
-           [open]="open"
-           (overlaySelected)="closeModal()">
+          [open]="open" 
+          (overlaySelected)="closeModal()">
   <cds-modal-header (closeSelect)="closeModal()">
-    <h3 cdsModalHeaderHeading
-        i18n>
-      Attach commands
-    </h3>
+    <h3 cdsModalHeaderHeading i18n>Attach commands</h3>
   </cds-modal-header>
-
   <div class="cds--modal-content">
-    <h5 class="fw-bold"
-        i18n>
-      Using Mount command
-    </h5>
-    <cd-code-block textWrap="true"
-                   [codes]="[mount]"></cd-code-block>
+    <form name="userForm" 
+          #formDir="ngForm" 
+          [formGroup]="userForm" 
+          novalidate>
+      <div class="form-item">
+        <cds-select
+          label="Ceph User"
+          for="user"
+          formControlName="user"
+          name="user"
+          id="user"
+          (change)="userChangeHandler()"
+          i18n
+          >Ceph User
```

Comment:
Why need this when we have `label` ?
```suggestion
          >
```

Reply:
Yeah correct, removed. Thanks!